core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -89,11 +89,15 @@ oppia.directive('outcomeDestinationEditor', [
               text: '(try again)'
             }];
 
+            if (StateEditorService.getInQuestionMode()) {
```

Comment:
Perhaps just call this at outset and store in local variable. Then in line 88 do `(questionModeEnabled ? null : currentStateName)`

Reply:
Done.